src/pageObjects/registrationPage/RegistrationPage.js

```diff
@@ -0,0 +1,52 @@
+import BasePage from "../BasePage.js";
+import { expect } from "@playwright/test";
+
+export default class RegistrationPage extends BasePage {
+  constructor(page) {
+    super(page, "/", page.locator("app-signup-modal"));
+
+    this.nameInput = this._container.locator("input#signupName");
+    this.lastNameInput = this._container.locator("input#signupLastName");
+    this.emailInput = this._container.locator("input#signupEmail");
+    this.passwordInput = this._container.locator("input#signupPassword");
+    this.repeatPasswordInput = this._container.locator(
+      "input#signupRepeatPassword",
+    );
+    this.registerButton = this._container.locator(
+      ".modal-footer button.btn-primary",
+    );
+    this.errorMessage = this._container.locator("div.invalid-feedback p");
+    this.signUpInput = this._container.locator(".form-group input");
+  }
+
+  async getErrorText() {
+    const actualErrorText = [];
+
+    for (const errorTextItem of await this.errorMessage.all()) {
+      actualErrorText.push(await errorTextItem.innerText());
+    }
+
+    return actualErrorText;
+  }
+
+  async clickAllMandatoryInputs() {
+    for (const input of await this.signUpInput.all()) {
+      await input.click();
+    }
+
+    await this.signUpInput.last().blur();
+  }
+
+  async checkCSS(item, property, value) {
+    await expect(item).toHaveCSS(property, value);
+  }
+
+  async fillTextAllInputs(text) {
```

Comment:
either pass object or multiple parameters, and pass particular value to particular input